lib/Account.php

```diff
@@ -1,11 +1,12 @@
 <?php
+namespace Stripe;
```

Comment:
I prefer the following formatting:

```php
<?php

namespace Stripe;

class ...
```

That style is more popular, for example, symfony use it.